scripts/ci/validatecode.ps1

```diff
@@ -38,10 +38,28 @@ function CheckBooLang(
     return $false
 }
 
-function CheckFile(
+function CheckCustomProfile(
+    [string]$FileName,
+    [string[]]$FileContent,
+    [int]$LineNumber
+) {
+    <#
+    .SYNOPSIS
+        Checks if the given profile is marked as a custom profile.
+        Returns true if such a reference exists.
+    #>
+    if ($FileName -notmatch "Examples" -and $FileContent[$LineNumber] -match "isCustomProfile: 1") {
```

Comment:
Do we care about profiles in the examples folders?

Reply:
@Kurtis, I'd say no.